src/command/remark/NextForecastByCommand.ts

```diff
@@ -0,0 +1,55 @@
+import { format, _ } from "commons/i18n";
+import { UnexpectedParseError } from "commons/errors";
+import { IBaseRemark, RemarkType, Remark } from "../remark";
+import { Command } from "./Command";
+
+export interface INextForecastByRemark extends IBaseRemark {
+  type: RemarkType.NextForecastBy;
+
+  day: number;
+  hour: number;
+  minute: number;
+}
+
+export interface INextForecastByRemarkDated extends INextForecastByRemark {
+  type: RemarkType.NextForecastBy;
+
+  date: Date;
+}
+
+export class NextForecastByCommand extends Command {
+  #regex = /^NXT FCST BY (\d{2})(\d{2})(\d{2})Z/;
+
+  canParse(code: string): boolean {
+    return this.#regex.test(code);
+  }
+
+  execute(code: string, remark: Remark[]): [string, Remark[]] {
+    const matches = code.match(this.#regex);
+
+    if (!matches) throw new UnexpectedParseError("Match not found");
+
+    const day = +matches[1];
+    const hour = matches[2];
```

Comment:
Nit: consistently `+` here or at `remark.push` for day/hour/minute

Reply:
I actually did this on purpose here as for the `description` string I want single digit day (e.g. `1` rather `01` while I do want the double digit hour and minute. Or should I add the `+` to both description and the `push` in this case?

Reply:
I see! Feel free to leave it as is then.